packages/nimble-components/src/table-column/icon/tests/table-column-icon.stories.ts

```diff
@@ -0,0 +1,153 @@
+import { html, ref } from '@microsoft/fast-element';
+import type { Meta, StoryObj } from '@storybook/html';
+import {
+    createUserSelectedThemeStory,
+    usageWarning
+} from '../../../utilities/tests/storybook';
+import { tableTag } from '../../../table';
+import { tableColumnIconTag } from '..';
+import {
+    SharedTableArgs,
+    sharedTableArgTypes,
+    sharedTableArgs
+} from '../../base/tests/table-column-stories-utils';
+import { iconXmarkTag } from '../../../icons/xmark';
+import { tableColumnTextTag } from '../../text';
+import { iconCheckLargeTag } from '../../../icons/check-large';
+import { iconQuestionTag } from '../../../icons/question';
+import { mappingIconTag } from '../../../mapping/icon';
+
+const simpleData = [
+    {
+        firstName: 'Ralph',
+        lastName: 'Wiggum',
+        status: 'fail',
+        isChild: true
+    },
+    {
+        firstName: 'Marge',
+        lastName: 'Simpson',
+        status: 'success',
+        isChild: false
+    },
+    {
+        firstName: 'Homer',
+        lastName: 'Simpson',
+        status: 'success',
+        isChild: false
+    },
+    {
+        firstName: 'Bart',
+        lastName: 'Simpson',
+        status: 'success',
+        isChild: true
+    },
+    {
+        firstName: 'Greg',
+        lastName: '',
+        status: 'foo',
+        isChild: true
+    },
+    {
+        firstName: 'Frank',
+        lastName: '',
+        status: 'bar',
+        isChild: false
+    }
+];
+
+const overviewText = `This page contains information about the types of columns that can be displayed in a \`nimble-table\`.
```

Comment:
I hope we don't have to duplicate this overview text in every column type's story. (I see that it does exist in both the text and anchor stories already). If you delete this does storybook autodocs will find it from one of the other stories to put it in the Docs page? 

Eventually this will be moot if we create one or more MDX pages instead of autodocs for this folder.

Reply:
This is moot now that I've separated the table column stories out, so there is no general column type overview anywhere.